### ISSUE TITLE
Improve default shopp templates responsiveness

### ISSUE THREAD

**Issue by ben72** [ingenesis/shopp#3186]
This is something I always have to fix when setting up a new Shopp install.
Improving this would give a much better user experience as a lot of new installs will just use the defaults.

It's especially a problem in cart.php and summary.php.

See this screenshot of the cart page using twentyfifteen:
https://www.dropbox.com/s/8zp8n8msbb4zcxz/Selection_092.jpg?dl=0

It's the same on large screens with this theme and on mobiles it's a problem with almost any theme.

**Comment by ben72** [comment]
Yeah I think it is, this one has some more information though so maybe close the other one?

**Comment by WazzaJB** [comment]
Agreed :)

Expanding on this too though, I think there is a lot of room for improvement within the default templates. I can't provide an extensive list yet because we don't use it at all - we have our own rolled out at Lightbox (which wouldn't however be to everyones taste - and is built purely as a foundation for integration with our bespoke themes).

**Comment by crunnells** [comment]
I've actually been working on this a bit this year for a bunch of client sites. It's tough to make tables responsive, but I think I have a decent solution for it. If nobody else wants to take a go at it, I'll take responsibility for this issue.

**Comment by ben72** [comment]
This is an important fix and something that should really work out of the box! I've fixed it for several sites as well. @crunnells I can verify your solution when you have a PR for it.

I think this is mostly what I do, haven't tested it now though so it might involve some other changes.
```
/* fix for cart page on small screen sizes */
@media screen and (max-width: 535px) {
	#shopp #cart select {
		max-width: 70px;
	}
	#shopp #cart .money, #shopp #cart .totals th, #shopp #cart .buttons td {
		white-space: normal; 
		font-size: 12px;
	}
}
```

**Comment by crunnells** [comment]
:thumbsup:
